### DISCUSSION TITLE
Abillity to mark beatmaps as hidden and have them not show in beatmap list

### DISCUSSION THREAD

**Discussion by bbb651** [ppy/osu#14530] [Ideas]
I often play multiplayer and download beatmaps to play them in multiplayer, and I don't want to delete them because I play them in multiplayer, but I don't intend to play them in singleplayer and I don't want to see them in the beatmap list.

I suggest a new entry in the context menu of beatmaps
![image](https://user-images.githubusercontent.com/53972231/131153290-4edc924f-12c3-42a9-8f2a-85e72685340b.png)
clicking it would toggle the beatmap being hidden, which would hide it from the beatmap list, unless "hidden beatmaps" is selected
![image](https://user-images.githubusercontent.com/53972231/131154174-b829f6c5-032d-44c2-ab7d-8297be48398b.png)
In addition to that, there'll be an option in the settings "auto-hide beatmaps downloaded in multiplayer" which would automatically hide beatmaps when downloaded while in multiplayer.

If this feature is wanted I can try to make this and submit a pull request.

**Comment by frenzibyte** [comment]
Hiding difficulties is an existing functionality in osu!lazer, what it's missing right now is the ability to hide a full beatmap, and a "hidden beatmaps" collection.

Not that I'm with this being a thing, but just noting.

**Comment by peppy** [comment]
It sounds like you want to manage a collection yourself, something like "my favourite maps" rather than hiding every other map? Otherwise you'd have to go through the full list and hide after each multiplayer session anyway..
